.github/ISSUE_TEMPLATE/blog_post.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Blog Post
+about: Suggestion for a blog post for medium.com/zowe
+title: ''
+labels: blog
+assignees: Joe-Winchester, Sujay Solomon
+
+---
+
+Thank you for taking the time to submit a blog post proposal.
+
+Please reference [Zowe Blog Guidelines](../../Blogging/blog_guidelines.md).
+
```

Comment:
Love this idea! In fact, @xrs888 has built a Kanban board for tracking Zowe Medium blog posts at https://github.com/openmainframeproject/foundation/projects/2. Thinking we could add this issue template to coordinate there to make things all in one stop - let me know your thoughts.

Reply:
@jmertic - yes this would be great to have the git issue template be tied to the Kanban board on the OMP repo.  I've not worked with github Kanban boards before, how do cards get added to the "New Blog Post" column ?  Would you and others be OK with all new blog post issues being in the OMP/foundation repo or would that clutter it ?

Reply:
I think it's fine @Joe-Winchester - probably will make it easier for us to track all our outstanding tasks. 

Adding items is something Chris can do as new issues are created ( and maybe even something we could do with a GH action ). We'd likely want to make a tag to call out these Medium posts and an ISSUE TEMPLATE to help streamline the process.

Sound good?